src/Common/Updater.php

```diff
@@ -57,6 +57,24 @@ public function __construct( $current_version = null ) {
 		$this->current_version = ! empty( $current_version ) ? $current_version : Tribe__Main::VERSION;
 	}
 
+	/**
+	 * Allows setting the version we are using for running tasks.
+	 * 
+	 * @param string $version
+	 */
+	public function set_version( string $version ) {
+		$this->current_version = $version;
+	}
+
+	/**
+	 * Gets the current version we are using for running tasks.
+	 * 
+	 * @return string The current version we are using.
+	 */
```

Comment:
```suggestion
	/**
	 * Gets the current version we are using for running tasks.
	 * 
	 * @since TBD
	 *
	 * @return string The current version we are using.
	 */
```